### PR TITLE
#169257352 user role setting

### PR DESCRIPTION
**Pull request by nshaphan**
## What does this PR do?
It allows a super administrator to assign roles to users.

#### Description of Task to be completed?
- [x] create all users list component
- [x] create a pop-up component for assigning the role
- [x] Fixes Navbar to adapt on user role changes

#### How should this be manually tested?
- Clone this repo
- switch to `ft-user-role-settings-169257352` branch
- Run npm i
- Run npm run watch
- Log in as a super administrator
- Click `Manage Users` Menu on Navbar to view all users
- Click on any user to assign a role

#### Any background context you want to provide?
If you don't have any user with a `super_administrator` role, access users database table and add a user with that role.

#### What are the relevant pivotal tracker stories?
[#169257352](https://www.pivotaltracker.com/story/show/169257352)
[#170752254](https://www.pivotaltracker.com/story/show/170752254)

#### Screenshots (if appropriate)

Users' list
---------------
image

Assign Role Popup
--------------------------
image

#### Questions:
N/A

### CODE REVIEW

File: src/assets/scss/components/_assignRole.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.assign-role {
+  label {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 1rem;
+    line-height: 0.9375rem;
+    color: #1F2041;
+  }
+}
```
Comment (Billmike) [inline src/assets/scss/components/_assignRole.scss]: Add new line here

File: src/assets/scss/components/_content.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@use "../utilities/variables";
+
+.main-content {
+  margin-top: 7rem;
+}
+
+.drop-shadow {
+  box-shadow: 0 0.625rem 1.875rem 0 variables.$color-box-shadow;
+}
```
Comment (Billmike) [inline src/assets/scss/components/_content.scss]: Add new line here

File: src/assets/scss/components/_usersList.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.users-list {
+  .item-user {
+    &:hover {
+      background-color: #f8f9fa;
+    }
+  }
+}
```
Comment (Billmike) [inline src/assets/scss/components/_usersList.scss]: New line here

File: src/assets/scss/components/_assignRole.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.assign-role {
+  label {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 1rem;
+    line-height: 0.9375rem;
+    color: #1F2041;
```
Comment (Billmike) [inline src/assets/scss/components/_assignRole.scss]: Let's add this to our list of colors

File: src/assets/scss/components/_modal.scss [hunk 5]
```diff
@@ -0,0 +1,80 @@
+.popup-view-container {
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  position: relative;
+  padding-top: 50px;
```
Comment (Billmike) [inline src/assets/scss/components/_modal.scss]: Use `rem` not `px`

File: src/assets/scss/components/_usersList.scss [hunk 6]
```diff
@@ -0,0 +1,7 @@
+.users-list {
+  .item-user {
+    &:hover {
+      background-color: #f8f9fa;
```
Comment (Billmike) [inline src/assets/scss/components/_usersList.scss]: Add this to the list of colors we have

**Comment by Billmike** [comment]
@nshaphan Resolve conflicts

**Comment by nshaphan** [comment]
> @nshaphan Resolve conflicts

Conflicts resolved

**Pull request merged**